src/BUILD

```diff
@@ -2,6 +2,15 @@ load("@rules_python//python:defs.bzl", "py_binary")
 load("@rules_python//python:defs.bzl", "py_test")
 load("@deps//:requirements.bzl", "requirement")
 
+exports_files(
+    ["data/both_t_data.csv",
+     "data/lexicons.csv",
+     "pickles/all_words.p",
+     "pickles/github_words.p",
+     "pickles/anger.p"],
```

Comment:
where are these pickles being used? I only see both_t_data being used.

Reply:
Oh, you are right. These pickles are used in `suite.py` which is under `src/`

Reply:
Hmm I can't get rid of them. Because, if I understand it correctly, you are executing from `main/xx.R` and that binary needs to know every data it will eventually access. I've tried taken them out but bazel complains.